lib/charms/layer/kubernetes_common.py

```diff
@@ -483,6 +483,7 @@ def generate_openstack_cloud_config():
         'password = {}'.format(openstack.password),
         'tenant-name = {}'.format(openstack.project_name),
         'domain-name = {}'.format(openstack.user_domain_name),
+        'tenant-domain-name = {}'.format(openstack.project_domain_name)
```

Comment:
Please add a comma to the end of this line.

Reply:
done